popup/js/model/options.js

```diff
@@ -55,6 +55,13 @@ export const defaultOptions = {
    */
   searchFuzzyness: 0.6,
 
+  /**
+   * Customized options for uFuzzy ('@leeoniya/ufuzzy') defaults and
+   * our builtin specifications. (See also
+   * https://github.com/leeoniya/uFuzzy/blob/main/src/uFuzzy.js#L9)
+   */
+  ufuzzyOptions: [],
```

Comment:
like stated above, I think it would be better to make this an object, like in uFuzzy itself:

```suggestion
  ufuzzyOptions: {},
```

Reply:
Yes, that's also what I preferred to use. Thanks!